people_detector/people_rect[] pr_vector
